tests/test_meta_tensor.py

```diff
@@ -422,21 +422,7 @@ def test_decollate(self, dtype):
 
     def test_str(self):
         t = MetaTensor([1.0], affine=torch.tensor(1), meta={"fname": "filename"})
-        s1 = str(t)
-        s2 = t.__repr__()
-        expected_out = (
-            "tensor([1.])\n"
-            + "Metadata\n"
-            + "\tfname: filename\n"
-            + "\taffine: 1\n"
-            + "\tspace: RAS\n"
-            + "\n"
-            + "Applied operations\n"
-            + "[]\n"
-            + "Is batch?: False"
-        )
-        for s in (s1, s2):
-            self.assertEqual(s, expected_out)
+        self.assertEqual(str(t), "tensor([1.])")
```

Comment:
thanks, I forgot that the tutorial relies on this to print some meta information https://colab.research.google.com/drive/1T4iAys-cC2qL80oJkIbAXAPlWNPwp4H7#scrollTo=hQv51ORy6_oV, so what would be your recommendation of print those? cc @rijobro

Reply:
In the tutorials we can do `print(repr(t))` to get the same behaviour.